lib/features/children/add_member/pages/failed_topup_bottomsheet.dart

```diff
@@ -38,9 +43,11 @@ class InsufficientFundsBottomsheet extends StatelessWidget {
         ],
       ),
       content: BodyMediumText(
-        'We couldn’t take the \$${amount.toStringAsFixed(0)} to top up your child’s wallet.\n\nDon\'t worry we will try again tomorrow.',
+        'We had trouble getting the \$$amount from your account for the $topup.\n\nNo worries, we will try again tomorrow!',
```

Comment:
Just to be sure. The amount shown here is the total amount? For example i create 2 children each with a top up of 5. Will $amount then be 10 or will it be 5?

Reply:
Andy decided to reword this without an amount so we dont have to bother with vpc, totals admin fee, it is a nightmare to phrase